### ISSUE TITLE
Failed to resolve: com.badlogicgames.gdxpay:gdx-pay-android-googlebilling:0.11.2

### ISSUE THREAD

**Issue by ProZhar** [libgdx/gdx-pay#177]
Failed to resolve: com.badlogicgames.gdxpay:gdx-pay-android-googlebilling:0.11.2 no found on gradle sync task.

```gradle
buildscript {
    repositories {
        google()
        jcenter()
        mavenLocal()
        mavenCentral()
        maven { url "https://jcenter.bintray.com" }
        maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
    }
    dependencies {
        classpath 'com.android.tools.build:gradle:3.1.4'
        classpath 'com.google.gms:google-services:4.0.1' // google-services plugin
    }
}

allprojects {
    apply plugin: "eclipse"
    apply plugin: "idea"

    version = '0.0.1'
    ext {
        appName = "Math"
//        gdxVersion = '1.9.9-SNAPSHOT'
        gdxVersion = '1.9.8'
        gpgsVersion = '15.0.1'

        multiDexVersion = '1.0.3'

        firebaseCoreVersion = '16.0.1'
        firebaseAdsVersion = '15.0.1'

        gdxPayVersion = '0.11.2'
    }

    repositories {
        google()
        jcenter()
        mavenLocal()
        mavenCentral()
        maven { url "https://jcenter.bintray.com" }
        maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
        maven { url "https://oss.sonatype.org/content/repositories/releases/" }
    }
}

project(":desktop") {
    apply plugin: "java"


    dependencies {
        implementation project(":core")
        implementation "com.badlogicgames.gdx:gdx-backend-lwjgl:$gdxVersion"
        implementation "com.badlogicgames.gdx:gdx-platform:$gdxVersion:natives-desktop"

    }
}

project(":android") {
    apply plugin: "android"

    configurations { natives }

    dependencies {
        implementation project(":core")
        implementation "com.badlogicgames.gdx:gdx-backend-android:$gdxVersion"
        natives "com.badlogicgames.gdx:gdx-platform:$gdxVersion:natives-armeabi"
        natives "com.badlogicgames.gdx:gdx-platform:$gdxVersion:natives-armeabi-v7a"
        natives "com.badlogicgames.gdx:gdx-platform:$gdxVersion:natives-arm64-v8a"
        natives "com.badlogicgames.gdx:gdx-platform:$gdxVersion:natives-x86"
        natives "com.badlogicgames.gdx:gdx-platform:$gdxVersion:natives-x86_64"

        implementation "com.android.support:multidex:$multiDexVersion"

        // GPGS
        implementation "com.google.android.gms:play-services-games:$gpgsVersion"
        implementation "com.google.android.gms:play-services-auth:$gpgsVersion"

        //Firebase
        implementation "com.google.firebase:firebase-core:$firebaseCoreVersion"
        implementation "com.google.firebase:firebase-ads:$firebaseAdsVersion"

        // ------ gdx-pay googlebilling --------
        implementation "com.badlogicgames.gdxpay:gdx-pay-android-googlebilling:$gdxPayVersion"
    }

    // ADD THIS AT THE BOTTOM
    apply plugin: 'com.google.gms.google-services'
}

project(":core") {
    apply plugin: "java"


    dependencies {
        implementation "com.badlogicgames.gdx:gdx:$gdxVersion"
        // ------ gdx-pay client --------
        implementation "com.badlogicgames.gdxpay:gdx-pay-client:$gdxPayVersion"
    }
}

tasks.eclipse.doLast {
    delete ".project"
}
```

[jcenter.bintray.com](https://jcenter.bintray.com/com/badlogicgames/gdxpay/) and [oss.sonatype.org (releases)](https://oss.sonatype.org/content/repositories/releases/com/badlogicgames/gdxpay/) do not have a gdx-pay-android-googlebilling directory

**Comment by keesvandieren** [comment]
gdx-pay-android-googlebilling has not yet been released. You can use 0.11.3-SNAPSHOT until it is released with repository: https://oss.sonatype.org/content/repositories/snapshots/ or use gdx-pay-android-googleplay

@MrStahlfelge are you already using googlebilling module in production? Do you consider it ready for release?

**Comment by MrStahlfelge** [comment]
Yes, I use it and the Amazon v0.11.3 in the production versions. Approved to release from my side. :-)

**Comment by keesvandieren** [comment]
Just released 0.11.3. Its already in https://oss.sonatype.org/content/repositories/releases, should be in Maven Central soon. Release notes here: https://github.com/libgdx/gdx-pay/releases/tag/v0.11.3
